src/test/scala/chisel3/tests/BundleLiteralsSpec.scala

```diff
@@ -36,35 +25,44 @@ class BundleLiteralSpec extends FlatSpec with ChiselScalatestTester {
       io.aOut := io.in.a
       io.bOut := io.in.b
     }) { c =>
-      c.io.in.poke(c.io.in.Lit(0.U, 1.U))
+      c.io.in.poke(c.io.in.cloneType.Lit(_.a -> 0.U, _.b -> 1.U))
```

Comment:
`cloneType` is a crappy, confusing, and informally-deprecated API. Please stop using it, or encouraging people to use it, or throwing it in new code. We can't actually mark it deprecated, because some legacy code makes heavy use of it, but new code should not use it.
Use `chiselTypeOf` instead.

Reply:
Agreed, I meant to ask you about this, but forgot about it. Fixed